Test/Mono.Cecil.Tests/BaseTestFixture.cs

```diff
@@ -119,19 +119,21 @@ abstract class TestCase {
 		public readonly bool ReadOnly;
 		public readonly Type SymbolReaderProvider;
 		public readonly Type SymbolWriterProvider;
+		public readonly BaseAssemblyResolver AssemblyResolver;
```

Comment:
Can you use IAssemblyResolver here and in the constructors instead of BaseAssemblyResolver?